addon/ng2/utilities/fs-promise.ts

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
This file is unnecessary. We already have `Promise.denodeify()` (look it up to see where to import it from).